src/shared/components/mutationMapper/MutationMapperConfig.ts

```diff
@@ -12,6 +16,8 @@ export interface IMutationMapperConfig {
     uniprot_id_url: string | null;
     ensembl_transcript_url: string | null;
     genomenexus_url: string | null;
+    genomenexus_client?: GenomeNexusAPI;
+    genomenexus_internal_client?: GenomeNexusAPIInternal;
```

Comment:
These should probably go into `IMutationMapperStoreConfig` with proper camel case names like `genomeNexusClient` and `genomeNexusInternalClient`. `IMutationMapperConfig` is simply a subset of `IServerConfig`. Any value that doesn't come directly from the server side shouldn't be included in this interface.

Reply:
discussed with @onursumer , remove this and add a constructor parameter in MutationMapperStore